pipelines/zendesk/requirements.txt

```diff
@@ -0,0 +1,2 @@
+zenpy==2.0.25
```

Comment:
should the version be fixed to this exact value. also please add it to our dev dependencies group as explained here:
https://github.com/dlt-hub/pipelines#pipeline-specific-dependencies